app/controllers/bids_controller.rb

```diff
@@ -130,58 +132,72 @@ def upload_rfq
 
 
     number_of_skipped_rows = 0
+    bid_quotation_for_transporter = BidQuotation.where(bid_id: @bid.id, transporter_id: transporter_id).first
+    if bid_quotation_for_transporter.nil?
+      bid_quotation = BidQuotation.create ({
+             bid_id: @bid.id,
+             bid_quotation_date: Date.today,
+             transporter_id: transporter_id
+               })
+    end
+
 
     (13..spreadsheet.last_row).each do |i|
```

Comment:
@nathnael what's special about the number 13? what does it stand for? would help to add comment to explain what it is.